src/Aspire.Hosting.Azure.AIFoundry/Internal/FoundryLocalManager.cs

```diff
@@ -0,0 +1,552 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
We should note that this file comes from https://github.com/microsoft/Foundry-Local/tree/main/sdk/cs/src and is copied locally until the NuGet package for the management SDK ships.